### DISCUSSION TITLE
uhhhhhhhh

### DISCUSSION THREAD

**Discussion by perngjiafu** [KadeArchive/Kade-Engine#2860] [Q&A] [answered]
hold note extends when i hold. how do i make it not like that

https://user-images.githubusercontent.com/54176781/154708557-f908c7bf-30c5-41b6-8732-52983c5055fe.mp4

.

**Comment by BoloVEVO** [comment, marked as answer]
Don't use Kade Engine 1.8 cuz is very buggy, use Kade-Engine-master instead (1.8.1 itw)

**Reply by perngjiafu** [reply]
w   h   e   r   e

**Reply by NotSorena** [reply]
@perngjiafu https://github.com/KadeDev/Kade-Engine/tree/master
here

**Reply by perngjiafu** [reply]
thxx
